cloudinit/distros/netbsd.py

```diff
@@ -21,10 +21,18 @@ class NetBSD(cloudinit.distros.bsd.BSD):
     """
 
     ci_sudoers_fn = '/usr/pkg/etc/sudoers.d/90-cloud-init-users'
-
     group_add_cmd_prefix = ["groupadd"]
-    pkg_cmd_install_prefix = ["pkg_add", "-U"]
-    pkg_cmd_remove_prefix = ['pkg_delete']
+
+    def __init__(self, name, cfg, paths):
+        super().__init__(name, cfg, paths)
+        if os.path.exists("/usr/pkg/bin/pkgin"):
+            self.pkg_cmd_install_prefix = ['pkgin', '-y', 'install']
+            self.pkg_cmd_remove_prefix = ['pkgin', '-y', 'remove']
+            self.pkg_cmd_remove_prefix = ['pkgin', '-y', 'update']
```

Comment:
copy paste error

we repeat the assignment variable `self.pkg_cmd_remove_prefix` with a different value

Reply:
oh! thanks.